src/hotspot/share/c1/c1_LIRGenerator.cpp

```diff
@@ -1420,7 +1420,9 @@ void LIRGenerator::do_getObjectSize(Intrinsic* x) {
 #else
   __ add(length_int, header_size, length_int);
   if (round_mask != 0) {
-    __ logical_and(length_int, LIR_OprFact::intConst(~round_mask), length_int);
+    // generates LIR_Const if mask fits instruction requirement
```

Comment:
I think the comment here is excessive, especially given other uses of `logical_and` that would probably get fixed too.